utils/forest_utils/src/net/http.rs

```diff
@@ -4,17 +4,25 @@
 use async_trait::async_trait;
 use hyper::{client::HttpConnector, Body};
 use hyper_rustls::{HttpsConnector, HttpsConnectorBuilder};
+use lazy_static::lazy_static;
 
-/// Constructs [hyper::Client] that supports both `http` and `https`.
-/// Note that only `http1` is supported.
-pub fn https_client() -> hyper::Client<HttpsConnector<HttpConnector>> {
-    hyper::Client::builder().build::<_, Body>(
+lazy_static! {
+    /// A default [hyper::Client]. It's imperative that the builder is only
+    /// called once, because fetching root certificates is expensive.
+    static ref CLIENT: hyper::Client<HttpsConnector<HttpConnector>> =
```

Comment:
Perhaps a `OnceCell` would be a bit cleaner. Not a huge deal, though.

Reply:
> Perhaps a `OnceCell` would be a bit cleaner. Not a huge deal, though.

Totally, but we are not on the 1.70 toolchain yet as far as I know.

Reply:
I think perhaps `Lazy` is a better option here, I'll incorporate that after raising a PR to bump the version.